_sass/base/_markdown.scss

```diff
@@ -29,12 +29,14 @@
   }
 
   // Lists
+  // Lists
```

Comment:
Ah yes, I'll remove it. Obviously as it's a comment it didn't actually do anything...